src/components/calcite-input/calcite-input.scss

```diff
@@ -119,87 +145,85 @@
 //focus
 :host textarea,
 :host input {
-  @include focus-style-base();
+  @apply focus-base;
 }
 :host textarea:focus,
 :host input:focus {
-  @include focus-style-inset();
+  @apply focus-inset;
 }
 
 // states
 :host {
   & input,
   & textarea {
-    color: var(--calcite-ui-text-1);
-    border: 1px solid var(--calcite-ui-border-1);
+    @apply text-color-1 
+      border 
+      border-solid 
+      border-color-input;
 
     &::placeholder,
     &:-ms-input-placeholder,
     &::-ms-input-placeholder {
-      color: var(--calcite-ui-text-3);
-      font-weight: 400;
+      @apply text-color-3 font-normal;
     }
   }
 
   & input:focus,
   & textarea:focus {
-    border-color: var(--calcite-ui-brand);
-    color: var(--calcite-ui-text-1);
+    @apply border-color-brand text-color-1;
   }
   & input[readonly],
   & textarea[readonly] {
-    background-color: var(--calcite-ui-background);
+    @apply bg-background;
   }
   & input[readonly]:focus,
   & textarea[readonly]:focus {
-    color: var(--calcite-ui-text-1);
+    @apply text-color-1;
   }
   & calcite-icon {
-    color: var(--calcite-ui-text-1);
+    @apply text-color-1;
   }
 }
 // input message slot
 :host slot:not[name="input-message"] {
-  display: block;
-  margin-bottom: $baseline/4;
-  color: var(--calcite-ui-text-2);
-  font-weight: 500;
+  @apply block 
+    mb-1 
+    text-color-2 
+    font-medium;
 }
 
 // icon
 :host([icon]) input {
-  padding-left: $baseline * 1.5;
+  @apply pl-8;
 }
 
 :host([dir="rtl"][icon]) input {
-  padding-right: $baseline * 1.5;
-  padding-left: $baseline/2;
+  @apply pr-8 pl-2;
 }
 
 :host([dir="rtl"][icon][scale="l"]) input {
-  padding-right: $baseline * 2;
-  padding-left: $baseline/2;
+  @apply pr-8 pl-2;
 }
 :host([icon][scale="l"]) input {
-  padding-left: $baseline * 2;
+  @apply pl-10;
 }
 //positioning wrapper for icon and loader
 .calcite-input-element-wrapper {
-  display: inline-flex;
-  flex: 1;
-  min-width: 20%;
-  position: relative;
-  order: 3;
+  @apply flex inline-flex relative order-3;
+  min-width: theme("width[1/5");
+  width: 100%;
```

Comment:
Can I do this? It didn't have this before on the input wrapper, but this is the only thing I could do to have it go full with on the date picker input (hex)

Reply:
removed, it created issues.